extensions/metrics-cluster-feature/Makefile

```diff
@@ -0,0 +1,5 @@
+install-deps:
```

Comment:
BTW is it requirement now for extensions to have such `Makefile` (and it's always looks the same as for mostly any other extension)?

Reply:
It's not a requirement. Lens Makefile just calls these automatically on `make build-extensions` (we could probably call plain `npm` commands).